bin/setup_alta

```diff
@@ -0,0 +1,18 @@
+sudo ip addr flush ganga
```

Comment:
let's put this in bin/setup_testing or something vaguely generic. It can still refer to alta internally (we'll make it parameteriable later), but want to try and get it in the right place to start!